README.md

```diff
@@ -33,14 +31,3 @@ Requires `nodejs` and `npm`.
 `make test`: Runs truffle tests.
 
 `make compile`: Compile contracts.
-
-# ChainSafe Security Policy
-
-## Reporting a Security Bug
-
-We take all security issues seriously, if you believe you have found a security issue within a ChainSafe
-project please notify us immediately. If an issue is confirmed, we will take all necessary precautions
-to ensure a statement and patch release is made in a timely manner.
-
-Please email us a description of the flaw and any related information (e.g. reproduction steps, version) to
-[security at chainsafe dot io](mailto:security@chainsafe.io).
```

Comment:
Actually @nmlinaric has a good idea for this, we should revert these changes but use sygma dev mail. @P1sar what do you say about this? We can also create security@buildwithsygma.com and use that?

Reply:
Yes, sounds good, I will ask to create this email.

Reply:
we can leave this [dev@buildwithsygma.com](mailto:dev@buildwithsygma.com) for now